### ISSUE TITLE
Feature checks missing for dmabuf import

### ISSUE THREAD

**Issue by cherusk** [any1/wayvnc#24]
Out of sudden, after running it for some time, on sway 1.4, ran into that error:

> ERROR: Failed to initialise renderer

By recompiling without:

```
    if (gl_load_late_extensions() < 0)
        goto late_extension_failure;
```

in `render_init`got it working again.

**Comment by any1** [comment]
So which procedure was it that it failed to load?

**Comment by any1** [comment]
Actually, those are all pretty much of the same ilk. What changed? Did you do a system upgrade? Perhaps upgrade mesa? Which distro are you running?

**Comment by cherusk** [comment]
> So which procedure was it that it failed to load?
Didn't trace that down in **gl_load_late_extensions**.

**Comment by cherusk** [comment]
> What changed? Did you do a system upgrade? Perhaps upgrade mesa? Which distro are you running?

Other way round, I had to upgrade to fedora31, since sway1.4 was only packaged for that. In previous sway, no VNC support.

It ran on fedora31/sway1.4 for some time, then without proper explanation, this ERROR kicked in. Tried to reinstall relevant libs. Without success. As mentioned, compiling it out, worked and now it's running sufficiently.

Currently, according to debug log, am on:
`DEBUG: OpenGL ES 3.1 Mesa 19.2.8`

Which is latest fedora packaged one.
